packages/ckeditor5-ckbox/tests/ckboximageedit/ckboximageeditui.js

```diff
@@ -0,0 +1,135 @@
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/* global window, btoa */
+
+import { global } from '@ckeditor/ckeditor5-utils';
+import ClassicTestEditor from '@ckeditor/ckeditor5-core/tests/_utils/classictesteditor';
+import Paragraph from '@ckeditor/ckeditor5-paragraph/src/paragraph';
+import CloudServices from '@ckeditor/ckeditor5-cloud-services/src/cloudservices';
+import { Image } from '@ckeditor/ckeditor5-image';
+import { setData as setModelData } from '@ckeditor/ckeditor5-engine/src/dev-utils/model';
+import { ButtonView } from '@ckeditor/ckeditor5-ui';
+import CloudServicesCoreMock from '../_utils/cloudservicescoremock';
+import TokenMock from '@ckeditor/ckeditor5-cloud-services/tests/_utils/tokenmock';
+
+import CKBoxImageEditEditing from '../../src/ckboximageedit/ckboximageeditediting';
+import CKBoxImageEditUI from '../../src/ckboximageedit/ckboximageeditui';
+
+describe( 'CKBoxImageEditUI', () => {
+	let editor, model, element, button, command;
+
+	beforeEach( () => {
+		TokenMock.initialToken = [
+			// Header.
+			'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9',
+			// Payload.
+			btoa( JSON.stringify( { auth: { ckbox: { workspaces: [ 'workspace1' ] } } } ) ),
+			// Signature.
+			'signature'
+		].join( '.' );
+
+		window.CKBox = {
+			mountImageEditor: sinon.stub()
```

Comment:
This looks like not been restored. I'd suggest using `testUtils.createSinonSandbox();` at the top level describe block.

Reply:
Done.